orangecontrib/geo/widgets/owmap.py

```diff
@@ -263,12 +263,28 @@ def set_marker_color(self, attr, update=True):
                     if not np.isnan(min) else [])
             elif variable.is_discrete:
                 _values = np.asarray(self.data.domain[attr].values)
-                __values = self.data.get_column_view(variable)[0].astype(np.uint16)
+
+                encoded_values = self.data.get_column_view(variable)[0]
+                is_na = np.isnan(encoded_values)
+                contains_na = np.any(is_na)
+
+                # Add a NA value + gray color locally (does not change attribute properties)
+                if contains_na:
+                    _values = np.append(_values, "NA")
+                    used_colors = np.vstack((variable.colors, [186, 189, 182]))
```

Comment:
In Scatter Plot we use [128, 128, 128] for gray and it would be nice to be consistent with it.